docs/snippets/web-components/register-component-with-play-function.js.mdx

```diff
@@ -0,0 +1,41 @@
+```js
+// RegistrationForm.stories.js
+import { userEvent, within } from '@storybook/testing-library';
+
+import { RegistrationForm } from './RegistrationForm';
+
+export default {
+  title: 'RegistrationForm',
+  component: RegistrationForm,
```

Comment:
According to https://github.com/storybookjs/storybook/blob/next/code/renderers/web-components/src/render.ts#L17-L31 (as well as https://storybook.js.org/docs/7.0/web-components/api/csf#upgrading-from-csf-2-to-csf-3), the `web-components` renderer has a default/implicit render function. Which means all of these play function snippets (and probably many more) can be made "common" and use the approach used here: https://storybook.js.org/docs/7.0/react/writing-stories/parameters#story-parameters. Wdyt?

Reply:
Sounds good to me. If we could make it more platform agnostic as possible, we should. Less maintenance cost and less error-prone issues